packages/custom-webpack/src/custom-webpack-builder.ts

```diff
@@ -7,8 +7,11 @@ export const defaultWebpackConfigPath = 'webpack.config.js';
 
 export class CustomWebpackBuilder {
 	static buildWebpackConfig(root: Path, config: CustomWebpackBuilderConfig, baseWebpackConfig: Configuration): Configuration{
+		if(!config) {
```

Comment:
Not required if the default value is {}

Reply:
you mean make default value {} and that's it?
but it still find `defaultWebpackConfigPath`, which is what I don't hope